com.microsoft.mrtk.input/Interactors/InteractorVisuals/Reticle.prefab

```diff
@@ -11,12 +11,12 @@ GameObject:
   - component: {fileID: 3867428799850922014}
   - component: {fileID: 7409364463680313149}
   m_Layer: 0
-  m_Name: RingCursor
+  m_Name: Reticle
   m_TagString: Untagged
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 0
```

Comment:
Is there a reason for deactivating the prefab by default?

Reply:
Not present in editor, might not have been serialized properly

Reply:
I think it was originally deactivated to not show up awkwardly in edit mode. I'll deactivate the model on the rig by default

Reply:
Done